Box.V2/Models/BoxFile.cs

```diff
@@ -113,5 +118,29 @@ public class BoxFile : BoxItem
         /// </summary>
         [JsonProperty(PropertyName = FieldRepresentations)]
         public BoxRepresentationCollection<BoxRepresentation> Representations { get; protected set; }
+
+        /// <summary>
+        /// The date when the file will be automatically deleted due to item expiration settings.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldExpiresAt)]
+        public DateTime? ExpiresAt { get; protected set; }
+
+        /// <summary>
+        /// The set of allowed roles for collaborators invited to this file.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAllowedInviteeRoles)]
+        public List<string> AllowedInviteeRoles { get; protected set; }
+
+        /// <summary>
+        /// Whether the item has collaborations or not.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldHasCollaborations)]
+        public bool? HasCollaborations { get; protected set; }
+
+        /// <summary>
+        /// Whether the item is owned by an entity external to the user's enterprise.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldIsExternallyOwned)]
+        public bool? IsExternallyOwned { get; protected set; }
```

Comment:
Should this be `Boolean?`

Reply:
In C#, `bool` is just an alias for `Boolean` — `bool?` is correct and allows a `null` value.